README.rst

```diff
@@ -37,6 +37,15 @@ Code Beiträge
 Änderungsvorschläge (sowohl an das Datenmodell als auch an die Implementierung in Python) können direkt als Code in Form von Pull Requests eingereicht werden.
 Details dazu finden sich im `Contribution Guide`_.
 
+Nutzung als Python Library
+==========================
+In Python kann diese Library als Paket installiert werden:
+
+.. code-block::
+
+       pip install bo4e
```

Comment:
Brauche ich dazu einen neuen Workspace? Wenn ja welchen?

Reply:
Du brauchst nur eine virtuelle Umgebung in de Python >=3.10 installiert ist